platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml

```diff
@@ -47,7 +47,8 @@ kind: Host
 metadata:
   name: {{ .Release.Name }}-nms
 spec:
-  hostname: cenm-nms.{{ .Values.global.proxy.externalUrlSuffix }}
+  # hostname: cenm-nms.{{ .Values.global.proxy.externalUrlSuffix }}
+  hostname: {{ .Values.global.proxy.prefix }}-nms.{{ .Values.global.proxy.externalUrlSuffix }}
```

Comment:
prefix which should match the cenm release name should be non global value

Reply:
Its a non global value as it is coming from networkmap.tpl (  .Values.global.proxy.prefix )